images/macos/provision/core/openjdk.sh

```diff
@@ -1,53 +1,96 @@
 #!/bin/bash -e -o pipefail
 source ~/utils/utils.sh
 
-JAVA_TOOLCACHE_PATH=$AGENT_TOOLSDIRECTORY/Java_Adopt_jdk
-
 createEnvironmentVariable() {
     local JAVA_VERSION=$1
-    local JAVA_PATH=$2
+    local ARCHITECTURE="x64"
```

Comment:
All the recent macOS versions don't support 32-bit apps, so this parameter can be hardcoded directly in the PATTERN

Reply:
@miketimofeev Done